packages/wrangler/src/__tests__/dev/remote-bindings-errors.test.ts

```diff
@@ -40,4 +52,58 @@ describe("errors during dev with remote bindings", () => {
 			"More than one account available but unable to select one in non-interactive mode."
 		);
 	});
+
+	it("errors triggered when establishing the remote proxy session (after it has been created) are appropriately surfaced", async () => {
+		vi.mocked(createPreviewSession).mockResolvedValue({
+			id: "test-session-id",
+			value: "test-session-value",
+			host: "test.workers.dev",
+			prewarmUrl: new URL("https://test.workers.dev/prewarm"),
+		});
+
+		vi.mocked(createWorkerPreview).mockImplementation(async () => {
+			throw new Error(
+				"R2 bucket 'non-existent-0' not found. Please use a different name and try again."
+			);
+		});
+
+		let thrownError: Error | undefined;
+
+		try {
+			await startRemoteProxySession(
+				{
+					MY_R2: {
```

Comment:
if there is no relation with R2, can we remove that too?

Reply:
removed